doc/getting_started/install_vivado/index.md

```diff
@@ -2,101 +2,26 @@
-   Create a new account if you don't have one yet.
-
-2. Before you proceed ensure that the download didn't get corrupted by verifying the checksum.
-
-    ```console
-    $ sha512sum --check Xilinx_Unified_2020.2_1118_1232_Lin64.bin.digests
-    Xilinx_Unified_2020.2_1118_1232_Lin64.bin: OK
-    sha512sum: WARNING: 22 lines are improperly formatted
-    ```
-
-    If you see an "OK" after the downloaded file proceed to the next step. Otherwise delete the download and start over. (You can ignore the warning produced by `sha512sum`.)
-3. Run the graphical installer. It may warn you that a newer version is available; you can ignore the warning.
-
-    ```console
-    $ sh Xilinx_Unified_2020.2_1118_1232_Lin64.bin
-    ```
-
-4. Now you need to click through the installer.
-   Click "Next" on the first screen.
-
-   ![Vivado installation step 1](img/step1.png)
-
-5. Type in your Xilinx User ID (your email address) and the associated password.
-   Choose the "Download and Install Now" option.
-   Click "Next" to continue.
-
-   ![Vivado installation step 2](img/step2.png)
-
-6. Click all "I Agree" checkboxes, and click on "Next" to continue.
-
-   ![Vivado installation step 3](img/step3.png)
-
-7. Choose "Vivado", and click on "Next" to continue.
-
-   ![Vivado installation step 4](img/step4.png)
-
-8. Choose "Vivado HL Design Edition".
-   This is required to enable support for the Xilinx Kintex 7 XC7K410T FPGA device found on the ChipWhisperer CW310 board.
-   You'll need a commercial Vivado license for this FPGA device.
-   Without a valid license, you are still able to install the Vivado HL Design Edition but you'll only be able to work with devices supported by the free WebPACK license.
-   If you don't have a valid license and if you're only planning to work with devices supported by the free WebPACK license, you can also choose "Vivado HL WebPACK" instead.
-
-   ![Vivado installation step 5](img/step5.png)
-
-9. Choose the features to install.
-   You can restrict the features to the ones shown in the screenshot below.
-   Click "Next" to continue.
-
-   ![Vivado installation step 6](img/step6.png)
-
-10. Choose an installation location.
-    Any location which doesn't have a whitespace in its path and enough free space is fine.
-    We use `/tools` in our example, but a path in `/opt` or within the home directory works equally well.
-    Click "Next" to continue.
-
-    ![Vivado installation step 7](img/step7.png)
-
-11. Double-check the installation summary and click on "Install" to start the installation process.
+_**Vivado Version:** The recommendation is to use Vivado {{< tool_version "vivado" >}}._
 
-    ![Vivado installation step 8](img/step8.png)
+See Chapter 4 ("Download and Installation") of the [Vivado Design Suite User
+Guide](https://www.xilinx.com/content/dam/xilinx/support/documents/sw_manuals/xilinx2020_2/ug973-vivado-release-notes-install-license.pdf#namedDest=InstallingTheVivadoDesignSuiteTools)
```

Comment:
https://docs.xilinx.com/r/2020.2-English/ug973-vivado-release-notes-install-license/Installing-the-Vivado-Design-Suite-Tools will get you to the section you wanted. That link doesn't work for me in Chrome's PDF viewer.

Though... maybe you meant to start from the section you named instead? https://docs.xilinx.com/r/2020.2-English/ug973-vivado-release-notes-install-license/Download-and-Installation

Reply:
I did mean to start from the "Download and Installation" section. It contains the "Installing the Vivado Design Suite Tools" section, and includes instructions for downloading the installer.

Thanks for the better links, I don't know how to navigate their documentation and only found the PDF version.